Removal of a duplicate paragraph

dvukolov opened the pull request:
## Notice

- [x] I acknowledge and agree that, by checking this box and clicking “Submit Pull Request”:

- I submit this contribution under the [Apache 2.0 license](https://www.apache.org/licenses/LICENSE-2.0.txt) and represent that I am entitled to do so on behalf of myself, my employer, or relevant third parties, as applicable.
- I certify that (a) this contribution is my original creation and / or (b) to the extent it is not my original creation, I am authorised to submit this contribution on behalf of the original creator(s) or their licensees.
- I certify that the use of this contribution as authorised by the Apache 2.0 license does not violate the intellectual property rights of anyone else.

## Motivation and Context
Why was this PR created?

## How has this been tested?
What testing strategies have you used?

## Checklist

- [x] Read the [contributing](/CONTRIBUTING.md) guidelines
- [ ] Opened this PR as a 'Draft Pull Request' if it is work-in-progress
- [ ] Updated the documentation to reflect the code changes
- [ ] Added new entries to the `RELEASE.md` file
- [ ] Added tests to cover my changes
- [ ] Assigned myself to the PR
- [ ] Added `Type` label to the PR

deepyaman commented:
@dvukolov Thanks for the catch and fix!

yetudada commented:
Thanks for catching this @dvukolov! ⭐️ We want to merge this PR. We need to you to add your name to the latest `RELEASE.md` under "Thanks for supporting contributions" for Kedro 0.15.0.

dvukolov commented:
Thank you for your offer @yetudada! I've updated the latest RELEASE.md and also removed an extra newline in the template README file. Hope you don't mind me doing it within the same pull request.
